pkgs/by-name/pr/proton-ge-custom/package.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, stdenvNoCC
+, fetchurl
+, writeScript
+}:
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "proton-ge-custom";
+  version = "GE-Proton9-1";
+
+  src = fetchurl {
+    url = "https://github.com/GloriousEggroll/proton-ge-custom/releases/download/${finalAttrs.version}/${finalAttrs.version}.tar.gz";
+    hash = "sha256-wCIffeayOy3kEwmIKB7e+NrliuSpKXoVYC334fxVB3U=";
+  };
+
+  buildCommand = ''
+    runHook preBuild
+    mkdir -p $out/{bin,opt}
+    tar -C $out/opt --strip=1 -x -f $src
+    ln -s $out/opt/toolmanifest.vdf $out/bin/toolmanifest.vdf
+    install -Dm644 $out/opt/compatibilitytool.vdf $out/bin/compatibilitytool.vdf
+    sed -i "s#\"install_path\" \".\"#\"install_path\" \"$out\/opt\/\"#g"  $out/bin/compatibilitytool.vdf
+    runHook postBuild
+  '';
+
+  passthru.updateScript = writeScript "update-proton-ge" ''
```

Comment:
Does `nix-update-script` not work?

Reply:
You might want to take a look on the comments 2nd part here https://github.com/NixOS/nixpkgs/pull/294532#issuecomment-1987359650 if it's okay if in sich situations the update script fails, nix-update can be used

Reply:
In that case it'd be great to have that as an explanation as a comment.